docs/replication-retry.md

```diff
@@ -0,0 +1,137 @@
+# Replication Retry
+
+## Description
+
+This feature offers a way for users to monitor and retry failed replication
+operations. It enables users to retrieve a list of failed operations and retry
+specific replication operations.
+
+## Requirements
+
+* The listing of failed operations.
+* The retry of a specific failed operation.
+
+## Design
+
+A RESTful API will expose methods for users to list and retry failed operations.
+See [Explanations](#explanations) for more detail.
+
+## Definition of API
+
+* GET `/_/crr/failed/all`
```

Comment:
For PR#200, I refactored the routes to follow:
`/_/metrics/<extension>/<site>/<specific-metric>`

Can these routes be edited to follow the same design? Maybe something like:
`/_/retry/crr/all`
`/_/retry/crr/all/<bucket>/<key>/<versionId>`

Reply:
I think `/all` can be dropped as it's implied already by the route.